Yubico.YubiKey/src/Yubico/YubiKey/IYubiKeyDeviceInfo.cs

```diff
@@ -126,5 +143,15 @@ public interface IYubiKeyDeviceInfo
         /// Indicates if this device has temporarily disabled NFC.
         /// </summary>
         public bool IsNfcRestricted { get; }
+
+        /// <summary>
+        /// The chip number, if available, otherwise empty string.
+        /// </summary>
+        string PartNumber { get; }
+
+        /// <summary>
+        /// Whether or not pin complexity is enabled on the Yubikey.
```

Comment:
Can we link to the YubiKey Manual section that explains PIN complexity here? https://docs.yubico.com/hardware/yubikey/yk-tech-manual/5.7-firmware-specifics.html#pin-complexity

Reply:
yes